ssl/quic/quic_channel.c

```diff
@@ -113,6 +114,93 @@ static int gen_rand_conn_id(OSSL_LIB_CTX *libctx, size_t len, QUIC_CONN_ID *cid)
     return 1;
 }
 
+static unsigned long chan_reset_token_hash(
+        const QUIC_STATELESS_RESET_TOKEN *tok)
+{
+    unsigned long h;
+
+    assert(sizeof(h) <= sizeof(tok->token));
+    memcpy(&h, tok->token, sizeof(h));
+    return h;
+}
+
+static int chan_reset_token_cmp(const QUIC_STATELESS_RESET_TOKEN *t1,
+                                const QUIC_STATELESS_RESET_TOKEN *t2)
+{
+    return CRYPTO_memcmp(t1->token, t2->token, sizeof(t1->token));
+}
+
+static int reset_token_obfuscate(QUIC_STATELESS_RESET_TOKEN *out,
+                                 const unsigned char *in, size_t len)
+{
+    /*
+     * TODO(QUIC): update this to AES encrypt the token in ECB mode with a
```

Comment:
Yeah, IMO it is TODO(QUIC FUTURE) as this would allow doing the comparison with plain memcmp so it would be perhaps nice if we need to support more reset tokens per QUIC channel.